mdsuite/time_series/base.py

```diff
@@ -26,7 +26,6 @@
 """
```

Comment:
are these used anywhere? Should we bring this under the calculators at some stage?

Reply:
They are probably not used at the moment so we should look into either removing or moving them

Reply:
They are nice to have as we can compute a moving average on properties which is very useful for pressure, energy, temperature, and so on. I think we should consider how to move them into the code under run with something like a `MovingAverage` calculator or something.